### PR TITLE
Add package: nim 0.19.0

### PR DESCRIPTION
**Pull request by Epictek**
based on @its-pointless PR

### CODE REVIEW

File: packages/nim/build.sh [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	make LD=$CC uos=linux mycpu=$NIM_ARCH myos=android  -j $TERMUX_MAKE_PROCESSES useShPath=$TERMUX_PREFIX/bin/sh  
+	cp config/nim.cfg ../host-build/config
+	
+	nim --opt:size --define:termux -d:release --os:android --cpu:$NIM_ARCH  -t:-I/data/data/com.termux/files/usr/include -l:"-L/data/data/com.termux/files/usr/lib -landroid-glob" c koch.nim
+	cd dist/nimble/src
+	nim --define:termux -d:release --os:android --cpu:$NIM_ARCH  -t:-I/data/data/com.termux/files/usr/include -l:"-L/data/data/com.termux/files/usr/lib -landroid-glob" c nimble.nim
+}
+termux_step_make_install() {
+	./install.sh $TERMUX_PREFIX/lib
+	cp dist/nimble/src/nimble $TERMUX_PREFIX/lib/nim/bin
```
Comment (joakim-noah) [inline packages/nim/build.sh]: `nim`, not `nimble`? Otherwise you're copying the same file two lines below.

File: packages/nim/build.sh [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+	cp koch $TERMUX_PREFIX/lib/nim/bin
+	cp dist/nimble/src/nimble $TERMUX_PREFIX/lib/nim/bin 
+	ln -sf $TERMUX_PREFIX/lib/nim/bin/nim $TERMUX_PREFIX/bin
```
Comment (xeffyr) [inline packages/nim/build.sh]:
```suggestion
	ln -sf $TERMUX_PREFIX/lib/nim/bin/nim $TERMUX_PREFIX/bin/
```
It's better to have a trailing '/' here as `/bin` is a directory.

File: packages/nim/build.sh [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+	ln -sf $TERMUX_PREFIX/lib/nim/bin/koch $TERMUX_PREFIX/bin
```
Comment (xeffyr) [inline packages/nim/build.sh]:
```suggestion
	ln -sf $TERMUX_PREFIX/lib/nim/bin/koch $TERMUX_PREFIX/bin/
```

File: packages/nim/build.sh [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+	ln -sf $TERMUX_PREFIX/lib/nim/bin/nimble $TERMUX_PREFIX/bin
```
Comment (xeffyr) [inline packages/nim/build.sh]:
```suggestion
	ln -sf $TERMUX_PREFIX/lib/nim/bin/nimble $TERMUX_PREFIX/bin/
```

**Comment by joakim-noah** [comment]
@fornwall, would be good to finally get this in.

**Comment by Epictek** [comment]
Bump (sorry)

**Comment by xeffyr** [comment]
> Bump (sorry)

Bumped !

Package is now available in [unstable-packages](https://github.com/termux/unstable-packages) repository.
```
pkg install unstable-repo
pkg install nim
```

**Comment by fornwall** [comment]
Nice work! Nim is now available in the main Termux package repository.

**Pull request merged**